habitat_baselines/utils/gym_definitions.py

```diff
@@ -73,38 +74,53 @@ def _make_habitat_gym_env(
     return env
 
 
-# Generic supporting general configs
-register(
-    id="HabitatGym-v0",
-    entry_point="habitat_baselines.utils.gym_definitions:_make_habitat_gym_env",
-)
+def try_register(id_name, entry_point, kwargs):
+    if id_name in registry.env_specs:
```

Comment:
This should raise an error.

Reply:
Or at least a warning

Reply:
This will be re-run if the user imports this module multiple times. Re-registering more than once causes issues. I do not think we want any warning, since it just doesn't register if the user already has registered.

Reply:
I see. 
Please make it private then.